app/controllers/complaints_controller.rb

```diff
@@ -4,7 +4,7 @@ class ComplaintsController < ApplicationController
   before_action :check_pa11y_id, only: :show, if: -> { Rails.env.ci? }
 
   def index
-    api = ApiDelegator.use("hses", "issues", {user: session["user"]})
+    api = ApiDelegator.use("hses", "issues", {user: session["user"], page: params[:page]})
```

Comment:
[thought] this seems unnecessary for the scope of this PR, since there is a default keyword and we don't have it working yet, and feels like it could make limit whoever does the frontend. (although that'll probably be you so maybe it's fine) am i missing how this work?